### PR TITLE
mariadb: fix compilation with libcxx

### PR DESCRIPTION
**Pull request by neheb**
Missing header and pointless cast.

Refresh all patches.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

Maintainer: @miska 
Compile tested: powerpc

### CODE REVIEW

File: utils/mariadb/patches/210-libcxx.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+--- a/storage/innobase/handler/ha_innodb.cc
```
Comment (BKPepe) [inline utils/mariadb/patches/210-libcxx.patch:1]: Is this part of the upstream repository or not? If not, can you please send it to them?

Comment (neheb) [reply utils/mariadb/patches/210-libcxx.patch:1]: Sent the second one. But this one fails on libstdcpp. What a pain...

**Comment by neheb** [comment]
Converting this to draft until I can fix it.

**Comment by neheb** [comment]
libcxx was kicked to the curb. Closing.

**Pull request closed without merging**